src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/EditAndContinueTests.cs

```diff
@@ -229,7 +231,8 @@ class A : System.Attribute
                     SemanticEdit.Create(SemanticEditKind.Update, ctorC0, ctorC1),
                     SemanticEdit.Create(SemanticEditKind.Update, ctorD0, ctorD1),
                     SemanticEdit.Create(SemanticEditKind.Update, method0, method1),
-                    SemanticEdit.Create(SemanticEditKind.Update, a0, a1)));
+                    SemanticEdit.Create(SemanticEditKind.Update, a0, a1),
+                    SemanticEdit.Create(SemanticEditKind.Update, ctorA0, ctorA1)));
```

Comment:
Without this change, the nullable diagnostics below changed. Here's why: this is a bit of a contrived test. ENC doesn't actually support changing the default value of a parameter, but if it did, we'd need to include the method in the list of things that actually changed. With the change in this commit, we no longer complete or analyze the method body or parameter of the constructor (because it was not in the semantic edit list, and the ctor was present in compilation0). This caused the nullable warning of the default parameter change from appearing. Since this isn't a scenario we actually support we could just remove it from the baseline, but I took the more "correct" approach of including the ctor update in the list of semantic updates, as it would need to appear if the IDE actually supported a change like this.